build_versions.json

```diff
@@ -4,15 +4,15 @@
       "puppet_release": 7,
       "puppet_version": "7.28.0",
       "puppetdb_termini_version": "7.16.0",
-      "terraform_version": "1.7.1",
+      "terraform_version": "1.7.2",
       "pdk_version": "3.0.1.3",
       "bolt_version": "3.27.4"
     },
     {
       "puppet_release": 8,
       "puppet_version": "8.4.0",
       "puppetdb_termini_version": "8.3.0",
-      "terraform_version": "1.7.1",
+      "terraform_version": "1.7.2",
```

Comment:
in the long term it would be really great to have a scanner for this that informs us about updates. maybe renovate bot with a custom plugin :thinking: